Doc/library/enum.rst

```diff
@@ -994,6 +1014,9 @@ Supported ``_sunder_`` names
 
 - ``_missing_`` -- a lookup function used when a value is not found; may be
   overridden
+- ``_ignore_`` -- a list of names, either as a :func:`list` or a :func:`str`,
```

Comment:
Should it be added to the `version added` section just below this?

Reply:
Done.